common/pulp/common/constants.py

```diff
@@ -53,6 +53,7 @@
 CALL_STATES = (CALL_WAITING_STATE, CALL_SKIPPED_STATE, CALL_ACCEPTED_STATE, CALL_RUNNING_STATE,
                CALL_SUSPENDED_STATE, CALL_FINISHED_STATE, CALL_ERROR_STATE, CALL_CANCELED_STATE,
                CALL_TIMED_OUT_STATE)
+VALID_STATES = set(filter(lambda state: state != CALL_CANCELED_STATE, CALL_COMPLETE_STATES))
```

Comment:
this belongs in ``tasks.py``, because this collection of states is "valid" in the context of that code only.